WordPress/build.gradle

```diff
@@ -105,7 +105,7 @@ android {
             buildConfigField "long", "REMOTE_CONFIG_FETCH_INTERVAL", "3600"
             buildConfigField "boolean", "GUTENBERG_MENTIONS", "true"
             buildConfigField "boolean", "HOME_PAGE_PICKER", "false"
-            buildConfigField "boolean", "WP_STORIES_AVAILABLE", "false"
+            buildConfigField "boolean", "WP_STORIES_AVAILABLE", "true"
```

Comment:
I think we could simply delete this line, and then the feature flag would be set to true as per the `defaultConfig` block [some lines above](https://github.com/wordpress-mobile/WordPress-Android/pull/13459/files#diff-276cc53d974bad3212f8e1f709d32ea3f8d380c4dccb10419505d283fb41f2d7R76).